packages/fluentui/react-northstar/src/components/Avatar/Avatar.tsx

```diff
@@ -98,6 +103,15 @@ const Avatar: React.FC<WithAsProp<AvatarProps>> & FluentComponentStaticProps<Ava
           }),
       })}
       {!image &&
+        Box.create(icon, {
+          defaultProps: () =>
+            getA11Props('icon', {
+              title: name,
+              styles: resolvedStyles.icon,
+            }),
+        })}
+      {!image &&
+        !icon &&
         Label.create(label || {}, {
```

Comment:
Not related to your changes, but this is not cool as if I will pass there `null` the slot will be still rendered